docs/ndcubesequence.rst

```diff
@@ -408,41 +391,167 @@ dimensions.shape ``(<Quantity 3.0 pix>, <Quantity 3.0 pix>, <Quantity
 Note that any cube axis can be input.  A common axis need not be
 defined.
 
-Extracting Data Arrays
--------------------------
+Plotting
+========
+
+Since ndcube 2.0, `~ndcube.NDCubeSequence` does not provide plotting methods.
+The rationale is explained in `Issue #315 <https://github.com/sunpy/ndcube/issues/315>`
+in our GitHub repo.
+If you feel that `~ndcube.NDCubeSequence` should support plotting again,
+please read and comment on that issue telling us about your use case.
+Better still, let us know that you would like to work on the necessary tools to enable
+sequence plotting, let us know with a comment on that issue.
+
+However, you can still visualize the data in `~ndcube.NDCubeSequence` in a number of ways.
+You can slice out a single `~ndcube.NDCube` and use its `~ndcube.NDCube.plot` method.
+Or you can write your own mixin class to define the plotting methods.
+Or finally, you can extract the data and use the myriad of plotting packages available in
+the Python ecosystem.  Below, we will outline these latter two options in a little more
+detail.
+
+Writing Your Own NDCubeSequence Plot Mixin
```

Comment:
This feels quite advanced, especially for putting in the docs before the manual plotting step. Are our docs aimed at users or developers of downstream packages?

Reply:
I think it is you're right the the manual stuff should come first.  I think moving the Mixin section to last probably mitigates the audience.  As users will not have to read through it to get to what they need, while developers (who may also be users) will not get lost in the manual stuff before reaching the more advanced mixin section.